test/OrderBy.spec.js

```diff
@@ -0,0 +1,29 @@
+const expect = require('chai').expect;
+require('..')();
+describe('Array#prototype#Reverse', function () {
+
+    it('Should return sorted based on provided key for all same types', function () {
+        const inputSameType = [{ x: 3, y: 9 },{ x: 2, y: 4 }];
+        const expectedSameType = [{ x: 2, y: 4 }, { x: 3, y: 9 }];
+        const actualSameType = inputSameType.OrderBy(x);
```

Comment:
`x` is not defined. `keySelector` must be a function.

Reply:
Fixed